src/scenes/NurseryRouter/PlantingProgressTabContent.tsx

```diff
@@ -23,19 +26,60 @@ type PlantingProgressProps = {
 };
 
 export default function PlantingProgress({ reloadTracking }: PlantingProgressProps): JSX.Element {
+  const dispatch = useAppDispatch();
+  const { selectedOrganization } = useOrganization();
   const theme = useTheme();
   const { isMobile } = useDeviceInfo();
   const { activeLocale } = useLocalization();
+  const featureFlagProjects = isEnabled('Projects');
+
+  const projects = useAppSelector(selectProjects);
+
+  const [filters, setFilters] = useState<Record<string, SearchNodePayload>>({});
   const [search, setSearch] = useState<string>('');
-  const [filters, setFilters] = useState<Record<string, any>>({});
   const [filterOptions, setFilterOptions] = useState<FieldOptionsMap>({});
   const [activeView, setActiveView] = useState<View>(initialView);
   const [selectedPlantingSiteId, setSelectedPlantingSiteId] = useState<number>(-1);
-  const dispatch = useAppDispatch();
-  const { selectedOrganization } = useOrganization();
+
+  const getProjectName = useCallback(
+    (projectId: number) => (projects?.find((project: Project) => project.id === projectId) || {}).name || '',
+    [projects]
+  );
+
+  const featuredFilters: FeaturedFilterConfig[] = useMemo(
+    () =>
+      featureFlagProjects
+        ? [
+            {
+              field: 'project_id',
+              options: (projects || [])?.map((project: Project) => `${project.id}`),
+              searchNodeCreator: (values: (number | string | null)[]) => ({
+                field: 'project_id',
+                operation: 'field',
+                type: 'Exact',
+                values: values.map((value: number | string | null): string | null =>
+                  value === null ? value : `${value}`
+                ),
+              }),
+              label: strings.PROJECTS,
+              renderOption: (id: string | number) => getProjectName(Number(id)),
+              notPresentFilterShown: true,
+              notPresentFilterLabel: activeLocale ? strings.NO_PROJECT : '',
```

Comment:
suggestion, make line 51 as (featureFlagProjects && activeLocale) and remove the check here